src/exchange/apiService.ts

```diff
@@ -57,6 +61,8 @@ export class APIService {
       ...config,
       transformRequest: config?.isAuthenticationRequired
         ? this.transformRequest
+        : url == SERVICE_URLS.USER.AUTHORIZE
```

Comment:
can we do it by passing additional flag instead of hardcoding the endpoint here?